src/Printing.jl

```diff
@@ -657,11 +664,17 @@ end
 ########################################################################
 #################### Footer ############################################
 ########################################################################
-
+# Footer edited to produce double horizontal
 function foot(t::TablePrinter)
     @unpack table_type = t.params
     table_type == :ascii && return ""
-    table_type == :latex && return "\\bottomrule\n\\end{tabular}"
+    if border == :double
+        table_type == :latex && return "\\hline\\hline\n\\end{tabular}"
+    elseif border == :single
+        table_type == :latex && return "\\bottomrule\n\\end{tabular}"
+    elseif border == :none
+        table_type == :latex && return "\\bottomrule\n\\end{tabular}"
+        @warn("default border will change to double in future releases")
```

Comment:
You're missing an end statement here.  That's why the travis builds are failing.  

In general, you can catch some of these mistakes locally by running `] test TexTables` on your machine.  All of the tests should pass before we merge this pr.

Reply:
I fixed the error and ran the build on my local machine as you suggested. The tests all passed locally.